### PR TITLE
feat: added increase, decrease and change date functions

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
### Description

This PR adds three "date manipulation" functions. One to specifically increase a period of time to a date, another to specifically decrease a period of time of a date and a third one that executes the increase/decrease logic. The third function can be called by itself, increasing and decreasing periods on the same call

### CODE REVIEW

**Comment by marcosalves-bitg** [comment]
Guys, let me know if I have to change the package.json version

**Comment by mickaelnn** [comment]
if You need to use this, You will need to change the package.json to a new version...

About the tests failing, I think have something about the timezone to change

File: src/date/index.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+export interface Duration {
+  day?: number
+  month?: number
+  year?: number
+  hour?: number
+  minutes?: number
+  seconds?: number
+  milliseconds?: number
+}
```
Comment (Viserion77) [inline src/date/index.ts:9]: Poderia criar um arquivo separado para isto

File: src/date/index.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+export const increaseDate = (date: Date, duration: Duration): Date => {
+  date = changeDate(date, duration)
+  return date
+}
+
+export const decreaseDate = (date: Date, duration: Duration): Date => {
+  for (const type in duration) {
+    duration[type] = duration[type] * -1
+  }
+  date = changeDate(date, duration)
```
Comment (Viserion77) [inline src/date/index.ts:30]: parâmetros imutáveis plz

**Review by Viserion77 — CHANGES_REQUESTED** [review]

**Comment by mickaelnn** [comment]
Precisa dar um npm toda vez que muda versão, se não fizer tem risco de pagar Coca

File: src/date/index.ts [hunk 3]
```diff
@@ -24,10 +16,11 @@ export const increaseDate = (date: Date, duration: Duration): Date => {
...
 
 export const decreaseDate = (date: Date, duration: Duration): Date => {
+  const decreases = {}
   for (const type in duration) {
-    duration[type] = duration[type] * -1
+    decreases[type] = duration[type] * -1
   }
-  date = changeDate(date, duration)
+  date = changeDate(date, decreases)
   return date
```
Comment (Viserion77) [inline src/date/index.ts]:
```suggestion
  return changeDate(date, decreases)
```

**Review by Viserion77 — CHANGES_REQUESTED** [review]

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by mickaelnn — DISMISSED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
LGTM

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by alanneves — APPROVED** [review]
LGTM

**Pull request merged**